CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 * Harvesters will work better with an engineer on board. (Sir Mortimer)
 * Fixed another icons sometimes not displaying bug (PiezPiedPy) 
 * SSPX 2.5m Greenhouse now producing food at the expected rate. (theJesuit)
+* ResourceBalance run the pressurizing\depressurizing (old "equalize\venting"), it gives priority to habitats with crew(HaullyGames)
```

Comment:
Could you word this for users to understand what's changed, the changelog is mainly for the players not for us coders 😉 hope I'm not being a pain.

Something like ' Habitats equalize/venting function changed to pressurizing/depressurizing, crewed habitats have priority while multiple habitats are pressurizing. New AirPump process added blah blah etc.